lib/ClangImporter/ImportType.cpp

```diff
@@ -556,7 +556,7 @@ namespace {
       if (size > 4096)
         return Type();
       
-      SmallVector<TupleTypeElt, 8> elts{size, elementType};
+      SmallVector<TupleTypeElt, 8> elts{static_cast<size_t>(size), elementType};
```

Comment:
Without this, clang++ complains about integer narrowing, as `size` is a zero-extended `uint64_t` and SmallVector expects a `size_t` length:
```
/home/butta/.termux-build/swift/src/swift/lib/ClangImporter/ImportType.cpp:509:41:
error: non-constant-expression cannot be narrowed from type 'unsigned long long' to 'size_t'
(aka 'unsigned int') in initializer list [-Wc++11-narrowing]
```

Reply:
Yeah, I ran into this as well and came up with this.  I thought Id merged that, but apparently not.